way_to_home/tests/unittests/place/testview.py

```diff
@@ -0,0 +1,245 @@
+            'name': 'Дім',
+            'address': 'Широка 34, 79052',
+            'stop_id': None,
+            'user_id': 2
+        }
+
+        url = reverse('place', kwargs={'place_id': self.place.id})
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertJSONEqual(json.dumps(expected_data), json.loads(response.content))
+
+    def test_get_all_success(self):
+        """Method that tests the successful request to all user places"""
+        expected_data = [
+        {
+            'id': 12,
+            'longitude': "48.842601",
+            'latitude': "21.968423",
+            'name': 'Робота',
+            'address': 'Сяйво 24, 79052',
+            'stop_id': None,
+            'user_id': 2
+        },
+        {
+            'id': 11,
+            'longitude': "49.842601",
+            'latitude': "23.968448",
+            'name': 'Дім',
+            'address': 'Широка 34, 79052',
+            'stop_id': None,
+            'user_id': 2
+        }]
+
+        url = reverse('place', args=[])
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertJSONEqual(json.dumps(expected_data), json.loads(response.content))
+
+    def test_get_failed_wrong_id(self):
+        """Method that tests request to retrieve non existent object."""
+        url = reverse('place', args=[1501])
+        response = self.client.get(url)
+
+        self.assertEqual(response.status_code, 404)
+
+    def test_get_non_owner(self):
+        """Method that tests for request to retrieve non owner Place instance."""
+        another_user = CustomUser(id=101, email='another_user@mail.com', is_active=True)
+        another_user.set_password('qwerty12345')
+        another_user.save()
+        self.client.login(email='another_user@mail.com', password='qwerty12345')
+
+        url = reverse('place', kwargs={'place_id': self.place.id})
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 403)
+
+    def test_post_success(self):
+        """Method that tests the success post request for creating place."""
+
+        data = {
+            'id': 13,
+            'longitude': 12.842601,
+            'latitude': 23.968448,
+            'name': 'Дім',
+            'address': 'Суха 3, 79052',
+            'stop_id': None
+        }
+
+        url = reverse('place', args=[])
+        response = self.client.post(url, json.dumps(data), content_type='application/json')
+        response_dict = json.loads(response.content)
+        self.assertEqual(response.status_code, 201)
+        self.assertEqual(response_dict['user_id'], 2)
+        self.assertEqual(response_dict['longitude'], 12.842601)
+        self.assertEqual(response_dict['latitude'], 23.968448)
+        self.assertEqual(response_dict['name'], 'Дім')
+        self.assertEqual(response_dict['address'], 'Суха 3, 79052')
+        self.assertEqual(response_dict['stop_id'], None)
```

Comment:
Did you try to use `assertDictEqual()` instead of this. Or is it not worked?

Reply:
Changed to `assertDictEqual()` It's working